packages/web3torrent/src/components/share-list/share-file/ShareFile.tsx

```diff
@@ -12,7 +12,7 @@ const ShareFile: React.FC<ShareFileProps> = ({file, goTo}: ShareFileProps) => {
       <td className="other-cell">{file.length}Mb</td>
       <td className="other-cell">{file.numPeers}S</td>
       <td className="other-cell">{file.numPeers}P</td>
-      <td className="other-cell">${file.cost}</td>
+      {/* <td className="other-cell">${cost}</td>  TODO reinstate this cell when we have a wer-per-byte model implpemented*/}
```

Comment:
When we get there, we can compute the cost directly from the `file`. As it is, we need the number of pieces, which is a property of the `torrent`.